includes/admin/class-wc-rest-payments-settings-controller.php

```diff
@@ -29,25 +28,16 @@ class WC_REST_Payments_Settings_Controller extends WC_Payments_REST_Controller {
 	 */
 	private $wcpay_gateway;
 
-	/**
-	 * Instance of WC_Payment_Gateway_WCPay.
-	 *
-	 * @var Digital_Wallets_Payment_Gateway
-	 */
-	private $digital_wallets_gateway;
-
 	/**
 	 * WC_REST_Payments_Settings_Controller constructor.
 	 *
-	 * @param WC_Payments_API_Client          $api_client WC_Payments_API_Client instance.
-	 * @param WC_Payment_Gateway_WCPay        $wcpay_gateway WC_Payment_Gateway_WCPay instance.
-	 * @param Digital_Wallets_Payment_Gateway $digital_wallets_gateway Digital_Wallets_Payment_Gateway instance.
+	 * @param WC_Payments_API_Client   $api_client WC_Payments_API_Client instance.
+	 * @param WC_Payment_Gateway_WCPay $wcpay_gateway WC_Payment_Gateway_WCPay instance.
 	 */
-	public function __construct( WC_Payments_API_Client $api_client, WC_Payment_Gateway_WCPay $wcpay_gateway, $digital_wallets_gateway ) {
```

Comment:
The purpose of this PR: removing the dependency from `Digital_Wallets_Payment_Gateway` and ensure that the settings are backwards compatible with the "older" settings view.